### ISSUE TITLE
RocketLogger UI reports software is uncalibrated with converted calibration file

### ISSUE THREAD

**Issue by gemarcano** [ETHZ-TEC/RocketLogger#34]
I just upgraded my RocketLogger to v2.0, and used the converter script to convert the old calibration. When I connect to the RocketLogger over the web browser, it indicates that it's uncalibrated. Is there a way to check if the software is finding and reading the calibration file from the SD card? The logs in `/var/log/rocketlogger` don't show any errors.

I'm attaching my old and new calibration files, in case it helps. Let me know if you need more information, and I'd be happy to provide it.
[old_new_calibrations.zip](https://github.com/ETHZ-TEC/RocketLogger/files/7214483/old_new_calibrations.zip)

**Comment by lsigrist** [comment]
Thanks a lot for sharing in detail the issues you're running into during the upgrade!

I was able to reproduce the described behavior that is actually a bug. After a reboot (or hardware reset) the web interface wrongly pretends that the logger is not calibrated even with correctly deployed calibration data. This is resolved as soon as the first measurement is triggered and the measurement status being updated. Can you confirm to observe the same behavior?

In case this does not resolve your issue, you might want to check if the SD card is mounted correctly by logging in to the system via SSH and displaying the content of `/etc/mtab`, which should list:
```
/dev/mmcblk0p1 /media/sdcard ext4 rw,noatime,errors=remount-ro 0 0
/dev/mmcblk0p1 /home/rocketlogger/.config/rocketlogger ext4 rw,noatime,errors=remount-ro 0 0
/dev/mmcblk0p1 /var/log/rocketlogger ext4 rw,noatime,errors=remount-ro 0 0
/dev/mmcblk0p1 /home/rocketlogger/data ext4 rw,noatime,errors=remount-ro 0 0
```

Your converted calibration data look very reasonable, assuming you have accidentally suffixed the converted file with ".old" ;)

**Comment by gemarcano** [comment]
Oh, lol, yeah, I mixed up the naming in the zip (it's correct on the actual system). And yes, I can confirm that actually starting a collection updates the UI to say that this particular calibration was done in January of 2020, which I think sounds correct.

**Comment by lsigrist** [comment]
fix verified using 0c130ac
